lessons/simple-inputs-and-wcipeg/index.adoc

```diff
@@ -48,30 +48,19 @@ int main() {
 >>> The Sum is 90
 ....
 
-== WCIPEG
+== M3C/MOJ
 
-http://wcipeg.com/[WCIPEG] is a website that has a collection of
-coding problems from various computer science contests as well as a wiki
-filled with useful information about coding algorithms. It also features
-an automated judge that will test any submitted code against its
-problem. Users could submit their code to be tested by the judge.
-
-Only registered users are able to use the online judge on the site. To
-register, go to the http://wcipeg.com/[WCIPEG main page] and click on
-*_register an account_* on the top right corner. To join the Markville
-Secondary School Organization on WCIPEG, go to *_My Account_ > _Edit
-User Info_*, and select *_Markville Secondary School_* as your
-organization. *:))*
-
-== DMOJ
-
-https://dmoj.ca/[DMOJ] is a website similar to WCIPEG that has
+https://m3cjudge.tk/[Markville Coding Online Judge] is a website that has
```

Comment:
Should this be `http` instead of `https`?

Reply:
Oops yes, I have not set up an ssl certificate yet.